### DISCUSSION TITLE
Midnight - An elegant and modern Theme

### DISCUSSION THREAD

**Discussion by ttytm** [TriliumNext/Trilium#2397] [Show and tell]
Uploaded another theme.  This style is a bit like VS Code. Pleasant to use.
_Also cleaned up the code of my first uploaded theme yesterday._

Link: https://github.com/tobealive/midnight-theme-trillium

![Trilium Notes - Midnight_001](https://user-images.githubusercontent.com/34311583/143969152-b24a5fbb-f8d2-4eda-a499-bc48b8635e31.jpg)

**Comment by Jayy001** [comment]
This is an amazing theme! Just wondering, is it possible to use a background image on it instead of a set color? :)

**Comment by ttytm** [comment]
Hey,
yes, it's possible. The implementation depends a bit on the way you want it and what kind of background it should be. E.g. a pattern or a full image. The different sections in the theme (Dock, Sidebar, Content, etc.) are colored in different shadings to create some segmentation. So it to keep the themes feeling and prevent everything from just being smudged together when using a background image it would be necessary to work with alpha levels on the overlaying layers to keep some separation between the sections. Maybe you can give some further explanation or an example of how you want to have it. Then I can help you better 🙂️✌️.
